src/main/java/www/purple/mixxy/helpers/GoogleAuthResponse.java

```diff
@@ -0,0 +1,75 @@
+package www.purple.mixxy.helpers;
+
+public class GoogleAuthResponse {
+	private String id;
+	private String email;
+	private Boolean verified_email;
```

Comment:
Works too. We don't even need to keep track of this, I just need this to parse the Json to a Java object